tasks/tool.py

```diff
@@ -85,7 +89,9 @@ def get_stack_json_outputs(ctx: Context, full_stack_name: str) -> Any:
 
 
 def get_aws_wrapper(aws_account: str) -> str:
-    return f"aws-vault exec sso-{aws_account}-account-admin"
+    if tool.is_ci():
```

Comment:
I am wondering if we should add instead a command line flag named `with-aws-vault`? What do you think?